catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -91,30 +91,22 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ExportCommand.class);
 
+    private static final int PAGE_SIZE = 64;
+
     private static final SimpleDateFormat ISO_8601_DATE_FORMAT;
 
+    private static Supplier<String> fileNamer;
```

Comment:
❗️ It's fine to move the initialization of this into the static block (though I'm not sure I see the advantage of it), but it should still be `final`.